crates/solver/src/main.rs

```diff
@@ -616,6 +618,15 @@ async fn main() {
         liquidity_order_owners: args.shared.liquidity_order_owners.into_iter().collect(),
         fee_objective_scaling_factor: args.fee_objective_scaling_factor,
     };
+    let tenderly = {
+        if let Some(tenderly_url) = args.tenderly_url {
+            if let Some(tenderly_api_key) = args.tenderly_api_key.as_ref() {
+                TenderlyApi::new(tenderly_url, client.clone(), tenderly_api_key).ok();
+            }
+        }
```

Comment:
nit: This rightward drift is a little unfortunate. Looks like `tenderly_url` and `tenderly_api_key` only make sense together. Maybe they can be passed around as a single struct?
I missed that on the previous PR. 🙈

Reply:
Can also be refactored as:
```rust
let tenderly = args.tenderly_url.zip(args.tenderly_api_key).map(|(url, api_key)| {
    TenderlyApi::new(url, client.clone(), api_key).ok();
});
```

Reply:
Took @nlordell suggestion, tnx both!